Sprintfinity3902/Entities/Items/SwordHitboxItem.cs

```diff
@@ -0,0 +1,122 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using Sprintfinity3902.Interfaces;
+using Sprintfinity3902.Link;
+using Sprintfinity3902.SpriteFactories;
+using System;
+
+namespace Sprintfinity3902.Entities
+{
+    public class SwordHitboxItem : AbstractEntity
+    {
+        Player PlayerCharacter;
+        Boolean itemUse;
+        int itemUseCount;
+        IPlayerState firingState;
+        Vector2 currentRect;
+
+        public SwordHitboxItem(Vector2 position)
+        {
+            Position = position;
+            currentRect = new Vector2(7, 12);
+            itemUse = false;
+            itemUseCount = 0;
+        }
+
+        public Boolean getItemUse()
+        {
+            return itemUse;
+        }
+
+        public override void Draw(SpriteBatch spriteBatch)
+        {
+
+
+            /*
+            
+            
+            if (firingState != null)
```

Comment:
There are a lot of comments that I am seeing. Are these just placeholders, or can they be deleted?

Reply:
This is also my fault. I'll need to clean up this class. Its been something I was tweaking and since Casey grabbed my branch for the wall collision, this got dragged along.